tests/__init__.py

```diff
@@ -9,3 +18,26 @@ def load_fixture(name):
     )
     with open(path, 'r') as f:
         return json.loads(f.read())
+
+
+def compare_item_with_dict(item: Item, data: dict):
```

Comment:
A nitpick: what do you think about renaming it, to make it clear what's the output - that it returns True if item and dict are equal? E.g. item_equals_dict. Also, you can annotate the returned value as bool, for completeness: 

`def item_equals_dict(item: Item, data: dict) -> bool:`